source/command-monitoring/tests/find.json

```diff
@@ -0,0 +1,239 @@
+      "_id": 3,
+      "x": 33
+    },
+    {
+      "_id": 4,
+      "x": 44
+    },
+    {
+      "_id": 5,
+      "x": 55
+    }
+  ],
+  "tests": [
+    {
+      "description": "A successful find event with no options",
+      "operation": {
+        "name": "find",
+        "arguments": {
+          "filter": {
+            "_id": 1
+          }
+        }
+      },
+      "expectations": [
+        {
+          "command_started_event": {
+            "command": {
+              "find": "values",
+              "filter": {
+                "_id": 1
+              }
+            },
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        },
+        {
+          "command_succeeded_event": {
+            "reply": [
+              {
+                "_id": 1,
+                "x": 11
+              }
+            ],
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        }
+      ]
+    },
+    {
+      "description": "A successful find event with options",
+      "operation": {
+        "name": "find",
+        "arguments": {
+          "filter": {
+            "_id": 1
+          },
+          "sort": {
+            "_id": 1
+          },
+          "skip": 2,
+          "limit": 2
+        }
+      },
+      "expectations": [
+        {
+          "command_started_event": {
+            "command": {
+              "find": "values",
+              "filter": {
+                "_id": 1
+              },
+              "sort": {
+                "_id": 1
+              },
+              "skip": 2,
+              "limit": 2
+            },
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        },
+        {
+          "command_succeeded_event": {
+            "reply": [
+              {
+                "_id": 1,
+                "x": 11
+              }
+            ],
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        }
+      ]
+    },
+    {
+      "description": "A successful find event with a getmore",
+      "operation": {
+        "name": "find",
+        "arguments": {
+          "filter": {
+            "_id": {
+              "$gte": 1
+            }
+          },
+          "sort": {
+            "_id": 1
+          },
+          "batchSize": 3
+        }
+      },
+      "expectations": [
+        {
+          "command_started_event": {
+            "command": {
+              "find": "values",
+              "filter": {
+                "_id": {
+                  "$gte": 1
+                }
+              },
+              "sort": {
+                "_id": 1
+              },
+              "batchSize": 3
+            },
+            "command_name": "find",
+            "database_name": "command-monitoring-tests"
+          }
+        },
+        {
+          "command_succeeded_event": {
```

Comment:
Java has it because of where the level is. .NET did not have it, which is why we store state from the command started event that the command succeeded/failed event needs: 

Storing: https://github.com/craiggwilson/mongo-csharp-driver/blob/csharp1347/src/MongoDB.Driver.Core/Core/Connections/CommandEventHelper.cs#L503

Using: https://github.com/craiggwilson/mongo-csharp-driver/blob/csharp1347/src/MongoDB.Driver.Core/Core/Connections/CommandEventHelper.cs#L735.